staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -354,6 +354,12 @@ type KubeletConfiguration struct {
 	// Default: "None"
 	// +optional
 	CPUManagerPolicy string `json:"cpuManagerPolicy,omitempty"`
+	// singleProcessOOMKill, if set, will prevent the `memory.oom.group` flag from being set for container
```

Comment:
The comment is a bit misleading. singleProcessOOMKill must be set to true. It can be "set" to false and that would also be the same as not setting it.